tests/io/test_read_commandline.py

```diff
@@ -0,0 +1,48 @@
+import os
```

Comment:
We should have a module-level docstring here.

```suggestion
"""Tests for read_commandline."""
import os
```